src/Command/ProbeDispatcherCommand.php

```diff
@@ -175,8 +175,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->setUp($input);
 
         $this->logger->info('Fireping Dispatcher Started.');
-        $this->logger->info('Slave name is ' . getenv('SLAVE_NAME'));
-        $this->logger->info('Slave url is ' . getenv('SLAVE_URL'));
+        $this->logger->info('Slave name is ' . $_ENV['SLAVE_NAME']);
```

Comment:
This has the same issue as described in the review of `src/AlertDestination/Mail.php`.